doc/man3/CMS_EncryptedData_decrypt.pod

```diff
@@ -26,15 +26,16 @@ to and I<flags> is an optional set of flags.
 I<dcont> is used in the rare case where the encrypted content is detached. It
 will normally be set to NULL.
 
-The following flags can be passed in the B<flags> parameter.
+The following flags can be passed in the I<flags> parameter.
 
-If the B<CMS_TEXT> flag is set MIME headers for type B<text/plain> are deleted
-from the content. If the content is not of type B<text/plain> then an error is
+If the B<CMS_TEXT> flag is set MIME headers for type C<text/plain> are deleted
+from the content. If the content is not of type C<text/plain> then an error is
 returned.
 
 CMS_EnvelopedData_decrypt() decrypts, similarly to CMS_decrypt(3),
-a CMS EnvelopedData object I<env> using the symmetric key I<secret> or
-the private key of the recipient B<pkey> and its associated certificate B<cert>.
+a CMS EnvelopedData object I<env> using the symmetric key I<secret> if it
+is not NULL, otherwise the private key of the recipient I<pkey> and
+the associated certificate I<cert>, which is then recommended to provide.
```

Comment:
It's not clear to me what the intention of _which is then recommended to provide_ is.  Any chance of rewording or clarifying this?

Reply:
Re-phrased this, referring to the NOTES in the doc of `CMS_decrypt()`.